### PR TITLE
3.7.3 release

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 3.7.3 (2020-01-24)
+
+- Fixed the repeater component to show the active item when using async lists. [#107](https://github.com/blackbaud/skyux-lists/pull/107)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: to display active items...?

Comment (blackbaud-johnly) [reply CHANGELOG.md]: Based on looking at the issue that this addresses (https://github.com/blackbaud/skyux-lists/issues/63) and the description of the `activeIndex` property (https://developer.blackbaud.com/skyux/components/repeater#repeater-properties), I think that the change we want here is actually to change `show` to `highlight`. Does that sound right, @Blackbaud-AlexKingman?

Also based on the issue that this addresses (https://github.com/blackbaud/skyux-lists/issues/63), we should change `when using async lists` to `when async lists load`? In Josh's description of the actual behavior, he says that when he selects another item, the active index starts to work. So it seems like maybe the problem is just on loading. Does that make sense, or am I misunderstanding?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# 3.7.3 (2020-01-24)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: I think we ned to update the date to the current date before we merge.

File: CHANGELOG.md (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md]: Yeah good points all around. I'll make the change.

Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 2 above)
Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md]: Done.

**Pull request merged**